mage_ai/orchestration/db/models/oauth.py

```diff
@@ -227,10 +228,10 @@ def get_parent_access(self, entity) -> int:
         This method is used when a role does not have a permission for a specified entity. Then,
         we will go up the entity chain to see if there are permissions for parent entities.
         '''
-        if entity == Permission.Entity.PIPELINE:
-            return self.get_access(Permission.Entity.PROJECT, get_project_uuid())
-        elif entity == Permission.Entity.PROJECT:
```

Comment:
Do you also want to remove the Entity definition in Permission class?

Reply:
yes, let me remove that